spec.html

```diff
@@ -26701,12 +26701,16 @@
           1. Let _truncatedStringFiller_ be a new String value consisting of repeated concatenations of _filler_ truncated to length _fillLen_.
           1. Return a new String value computed by the concatenation of _S_ and _truncatedStringFiller_.
         </emu-alg>
-        <emu-note>The first argument _maxLength_ will be clamped such that it can be no smaller than the length of the *this* value.</emu-note>
-        <emu-note>The optional second argument _fillString_ defaults to *" "* (a String consisting of 0x0020 SPACE).</emu-note>
+        <emu-note>
+          <p>The first argument _maxLength_ will be clamped such that it can be no smaller than the length of the *this* value.</p>
```

Comment:
They aren't required in the sense that the styles are identical with and without them. However, ecmarkup can't just auto-wrap in p's because sometimes notes have multiple paragraphs. I could look to see if there are p tags inside, but decided it wasn't worth the effort at the time. You can open an issue on ecmarkup if you feel strongly!